index.html

```diff
@@ -9965,22 +10018,35 @@ <h4 id="Beast Shape - Small Animal">Beast Shape - Small Animal</h4>
             <tr>
                <td>Core
                </td>
+               <td>Ankle Bite
+               </td>
+               <td>Make a regular melee attack against a target. On a successful hit do
+                  Light+DEX poison damage.
+               </td>
+               <td>E, Melee
+               </td>
+               <td>2
+               </td>
+            </tr>
+            <tr>
+               <td>1
+               </td>
                <td>Poison Strike
                </td>
                <td>Make a regular melee attack against a target. On a successful hit do
-                  Medium+DEX poison damage and <a class="condition" data-tooltip-container>hobble</a>. 2 FORT to resist <a class="condition" data-tooltip-container>hobble</a>
+                  Light+DEX poison damage and <a class="condition" data-tooltip-container>paralyse</a>. 2 FORT to resist <a class="condition" data-tooltip-container>paralyse</a>
                </td>
                <td>E, Melee
                </td>
                <td>2
                </td>
             </tr>
             <tr>
-               <td>Core
+               <td>1
                </td>
                <td>Scamper
                </td>
-               <td>Move 50ft. Can climb upwards at normal speed
+               <td>Move 35ft. Can climb upwards at normal speed
```

Comment:
I feel like this can maybe be more, I know it does climb speed. Purely because 5ft extra just seems almost like being given half a penny, make it 40ft for a whole penny, please sir